pyproject.toml

```diff
@@ -77,6 +77,9 @@ dev = [
     "pytest-sugar>=0.9",
     "docker>=6",
 
+    # security
+    "pip-audit",
```

Comment:
I wonder if we actually need to install `pip-audit` or if we can just use the existing pre-commit hook?
https://github.com/pypa/pip-audit?tab=readme-ov-file#pre-commit-support